File: Two sum/twosum.py
# Twosum
def twosum(nums,target):
    d={}
    for i,num in enumerate(nums):
        if target-num in d:
            return [d[target-num],i]
        d[num]=i
    return []
nums=[2,7,11,15]
target=9
print(twosum(nums,target))

File: Two sum/twosum.js
// Two sum
function twosum(nums,target){
    dict={}
    for(let i=0;i<nums.length;i++){
        if(target-nums[i]  in dict){
            return [dict[target-nums[i]],i]
        }
        else{
            dict[nums[i]]=i
        
        }
    }
    return "No two sum solution"
}
console.log(twosum([2,7,11,15],9));